godot-core/src/builtin/plane.rs

```diff
@@ -115,6 +201,39 @@ impl Plane {
             || (self.normal.is_equal_approx(-other.normal) && is_equal_approx(self.d, -other.d))
     }
 
+    /// Returns `true` if the `self` `Plane` is finite by calling `is_finite` on `normal` and `d`. 
+    ///
+    /// _Godot equivalent: `Plane.is_finite()`_
+    #[inline]
+    pub fn is_finite(&self) -> bool {
+        self.normal.is_finite() && self.d.is_finite()
+    }
+
+    /// Returns `true` if `point` is located above the `self` `Plane`.
+    ///
+    /// _Godot equivalent: `Plane.is_point_over(Vector3 point)`_
+    #[inline]
+    pub fn is_point_over(&self, point: Vector3) -> bool {
+        self.normal.dot(point) > self.d
+    }
+
+    /// Returns a normalized copy of the `self` `Plane`.
+    ///
+    /// _Godot equivalent: `Plane.normalized()`_
+    #[inline]
+    pub fn normalized(&self) -> Self {
+	    self.normal.normalized();
+        self.clone()
+    }
```

Comment:
This implementation is just returning `self.clone()`.

The past-tense also indicates that a copy is generated (rather than mutating the object directly), so `self.normal.normalized()` has no effect.

You can use the `#[must_use]` attribute here, to make sure the caller gets a warning if they don't use the result.